test/qunit/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="../../dist/js-reporters.js"></script>
+    <link href="../../node_modules/qunitjs/qunit/qunit.css" rel="stylesheet" type="text/css"/>
+    <script src="../../node_modules/qunitjs/qunit/qunit.js"></script>
+    <script src="tests.js"></script>
+
+    <script>
+        var adapter = new JsReporters.QUnitAdapter();
```

Comment:
After some discussion with @fcarstens elsewhere: `runner` and `testRunner` (when the test context isn't obvious) should work. Once frameworks implementation the interface themselves, the adapters can be replaced, and we can refer to the frameworks object itself as the `runner`.